PyDynamic/identification/fit_filter.py

```diff
@@ -39,7 +39,7 @@ def fitIIR(Hvals, tau, w, E, Na, Nb):
 
     Parameters
     ----------
-        Hvals:   numpy array of frequency response values of shape (M,)
+        Hvals:   numpy array of (complex) frequency response values of shape (M,)
```

Comment:
Sorry for the question, but doesn't that mean that in the complex case somewhere in the dimension occurs a 2?

Reply:
Actually no, because numbers of complex numpy.dtype directly support the use of complex numbers in the entry of an array. So no need to manually define complex number from independent arrays of their real and imaginary parts.